_rules/SC1-1-1-image-accessible-name-is-placeholder-text.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Image accessibile name is placeholder text
+description: |
+	This rule checks for images that use generic placeholder text.
```

Comment:
Hum, I'm a bit confused :-) 

For example, for a rule that is based on the same failure (F30 #263), the description ask *what the alt should not be* :  "This rule checks that the accessible name of an image is not its file name".

In that rule (placeholder text), we are looking for an equivalent outcome: what the alt should not be. 

Proposal:

> This rule checks that the accessible name of an image is not a generic placeholder text